modules/combined/test/tests/thermal_elastic/tests

```diff
@@ -11,10 +11,8 @@
     prereq = test
   [../]
   [./derivatives]
-    type = 'PetscJacobianTester'
+    type = 'AnalyzeJacobian'
     input = 'derivatives.i'
-    ratio_tol = 8e-7
-    difference_tol = 1e-5
     method = '!DBG' # slow test
```

Comment:
I should probably remove this restriction. On my machine the dbg test takes about 18s, and only dbg is tested for modules.